gulpfile.ts

```diff
@@ -15,6 +15,8 @@ import rollupTypescript from 'rollup-plugin-typescript2';
 import { babel as rollupBabel } from '@rollup/plugin-babel';
```

Comment:
This file is a bit long, maybe it can be split.

Reply:
Create folders for build scripts?

Reply:
make a folders named `tools`

Reply:
keep gulpfile.ts in the root directory

Reply:
okay